web/src/lib/components/shared-components/control-app-bar.svelte

```diff
@@ -66,7 +66,7 @@
   let buttonClass = $derived(forceDark ? 'hover:text-immich-dark-gray' : undefined);
 </script>
 
-<div in:fly={{ y: 10, duration: 200 }} class="absolute top-0 w-full bg-transparent z-1">
+<div in:fly={{ y: 10, duration: 200 }} class="absolute top-0 w-full bg-transparent z-0">
```

Comment:
You need z-1 here, as otherwise the asset select control bar in an album is broken

Reply:
Fixed the page to allow Z stay 0, the only good Z is the dead Z

Reply:
<img width="225" alt="Screenshot 2025-05-19 at 12 36 34" src="https://github.com/user-attachments/assets/5ed33aa4-2409-44e3-a070-79dde241ca8f" />

Nope :D
I already tried very hard on this one, I think we just need to accept a `z-1` here